src/app/components/shell/welcome/welcome-content.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {useDialog} from '~/components/dialog/dialog';
+import cookie from '~/helpers/cookie';
+import { createStoreon } from 'storeon';
+import { StoreContext } from 'storeon/preact';
+import user from '~/pages/my-openstax/store/user';
+import useAccount from '~/pages/my-openstax/store/use-account';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import {useHistory} from 'react-router-dom';
+import './welcome.scss';
+
+function WalkthroughButtons({welcomeDone}) {
+    const history = useHistory();
+
+    function goToMyOpenStax() {
+        welcomeDone();
+        window.setTimeout(() => {
+            history.push('/');
+        }, 100);
+    }
+
+    function skipWalkthrough() {
+        cookie.setKey('walkthroughDone');
+        goToMyOpenStax();
+    }
+
+    function showWalkthrough() {
+        window.localStorage.showMyOpenStaxWalkthrough = true;
+        goToMyOpenStax();
+    }
```

Comment:
Keep in mind that safari won't allow localStorage in an incognito window, so don't do anything that depends on it.  I've also been using https://github.com/astoilkov/use-local-storage-state which is pretty nice